service/src/test/java/uk/gov/hmcts/reform/fpl/utils/CoreCaseDataStoreLoader.java

```diff
@@ -28,4 +28,9 @@ public static CallbackRequest callbackRequest() throws IOException {
         String response = ResourceReader.readString("core-case-data-store-api/callback-request.json");
         return mapper.readValue(response, CallbackRequest.class);
     }
+
+    public static CallbackRequest careOrderRequest() throws IOException {
```

Comment:
Consider using the CaseDataGeneratorHelper

Reply:
Will stick with this implementation for now as it gives us clean and concise tests, when adding next type of order will create a folder to store the .json files (we will have one for each order type)

Reply:
I like the solution but does it belong in CoreCaseDataStoreLoader.json?